README.md

```diff
@@ -103,8 +103,6 @@ or
 ./start_docker.sh -u
 ```
 
-If the requirements.txt has changed, Docker will rebuild with the new dependencies first.
```

Comment:
Can you add a line saying they'll need to pull the updated docker image as well, with the appropriate code